verticapy/utilities.py

```diff
@@ -67,42 +66,103 @@
 
 #
 # ---#
-def create_verticapy_schema(cursor=None):
+def create_table(
+    table_name: str,
+    dtype: dict,
+    schema: str = "",
+    temporary_table: bool = False,
+    temporary_local_table: bool = True,
+    genSQL: bool = False,
+):
     """
 ---------------------------------------------------------------------------
-Creates a schema named 'verticapy' used to store VerticaPy extended models.
+Creates a new table using the input columns names and data types.
 
 Parameters
 ----------
-cursor: DBcursor, optional
-    Vertica database cursor.
+table_name: str, optional
+    The final table name.
+dtype: dict
+    Dictionary of the user types. Each key represents a column name and each
+    value represents its data type. 
+    Example: {"age": "int", "name": "varchar"}
+schema: str, optional
+    Schema name.
+temporary_table: bool, optional
+    If set to True, a temporary table will be created.
+temporary_local_table: bool, optional
+    If set to True, a temporary local table will be created. The parameter 
+    'schema' must to be empty, otherwise this parameter is ignored.
+genSQL: bool, optional
+    If set to True, the SQL code to use to create the final table will be 
```

Comment:
"the SQL code to use to create" -> "the SQL code for creating"

Reply:
It will be corrected in the next commit.